lib/src/main/java/com/diffplug/spotless/sql/dbeaver/SQLTokenizedFormatter.java

```diff
@@ -409,10 +404,7 @@ private int insertReturnAndIndent(final List<FormatterToken> argList, final int
 						prevToken.getType() != TokenType.END) {
 					s.setCharAt(0, ' ');
 					s.setLength(1);
-
-					final String comment = token.getString();
-					final String withoutTrailingWhitespace = comment.replaceFirst("\\s*$", "");
-					token.setString(withoutTrailingWhitespace);
```

Comment:
kind of documentation, kind of coupling burden:

- https://github.com/pmd/pmd/issues/5770